server/db/getUser.js

```diff
@@ -0,0 +1,32 @@
+var client = require('./init.js');
+
+function getUser (id, callback) {
+
+    var hashKey = 'user:' + id;
+
+    client.hgetallAsync(hashKey)
+        .then((response) => {
+            console.log(1);
+            if (!response)
```

Comment:
Quick one, the if statement below has the open curly, but this one doesn't any reasons?